src/pylibsshext/channel.pyx

```diff
@@ -170,9 +183,35 @@ cdef class Channel:
 
         return result
 
+    def send_eof(self):
+        """Send EOF to the channel, this will close stdin."""
+        rc = libssh.ssh_channel_send_eof(self._libssh_channel)
+        if rc != libssh.SSH_OK:
+            raise LibsshChannelException("Failed to ssh_channel_send_eof: [%d]" % rc)
+
+    def send_signal(self, sig):
+        """
+        Send signal to the remote process.
+
+        @sig is a signal constant from ``signal``, e.g. ``signal.SIGUSR1``.
```

Comment:
Let's follow the native Sphinx param list style just like in other places:
```suggestion
        :param sig: a signal constant from ``signal``, e.g. ``signal.SIGUSR1``.
        :type sig: 
```

Alternatively, you could've used double backticks for `sig`. But the `@sig` syntax is not something that is typical for Python (I believe it comes from Java or JS?).

Reply:
doxygen, fixed